torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1040,6 +1040,21 @@ def sample_inputs_trace(self, device, dtype, requires_grad, **kwargs):
                                      requires_grad=requires_grad))),)
 
 
+def sample_inputs_renorm(self, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+    cases = (((S, S, S), (2, 1, 0.5)),
```

Comment:
Note: we might want to start preserving names (where possible) now that SampleInputs have a "name" attribute